omod/src/main/java/org/openmrs/module/uicommons/fragment/controller/GlobalPropertiesFragmentController.java

```diff
@@ -20,8 +20,7 @@ public SimpleObject get(@RequestParam("properties") String properties,
             if (gp == null) {
                 ret.put(propertyName, null);
             } else {
-                SimpleObject simple = new SimpleObject();
-                simple.put("propertyValue", gp.getPropertyValue());
```

Comment:
@wluyima as it stands before the changes a new object that is not used anywhere is created. I simply changed this to how it was supposed to be. It is a no brainer and I didn't create of course because this was a very quick fix I did.